tsl/test/expected/compression_bgw.out

```diff
@@ -185,3 +186,26 @@ order by chunk_name;
  test_table_int  | _timescaledb_internal._hyper_3_6_chunk | 24 kB                    | 16 kB
 (2 rows)
 
+--TEST 8
+--hypertable owner lacks permission to start background worker
+\c :TEST_DBNAME :ROLE_SUPERUSER
```

Comment:
tests usually connect wit DEFAULT_PERM_ROLE_USER unless super user privileges are required. I don't think it is needed in this case. Could you please use one of the regular users instead?